src/style.css

```diff
@@ -1,6 +1,16 @@
 * {
   list-style: none;
   padding: 0;
+  font-family: sans-serif;
+}
+
+html {
+  background-image: url(http://www.space-art.co.uk/img/full-sized/milky-way-art.jpg);
```

Comment:
- [ ] [OPTIONAL]: Right now, all I have is a plain white background, you should consider double-checking it!

![image](https://user-images.githubusercontent.com/66967274/147244962-2bc207fa-b779-47d5-a6ae-21fc4fc560d6.png)